src/Tools.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <string>
+
+void StartBotProcess(const std::string& CommandLine);
+
+void SleepFor(int seconds);
+
+void KillSc2Process(unsigned long pid);
+
+int MoveFile(const char* lpExistingFileName, const char* lpNewFileName);
```

Comment:
There is no MoveFile in ToolsWindows.h while it should be one.

Reply:
Yes - the MoveFile method is implemented in Windows.h.
It should probably be commented in the ToolsWindows.h file so this doesn't confuse people, because, admittedly, it's not very intuitive.

Reply:
This is correct, we should probably avoid that and create a proxy function in ToolsWindsows so to keep consistancy.  We could rename it to avoid confusion.  Ideally, after this patch, we should only be including Windows.h from the ToolsWindows.cpp include, we should avoid platform specific code in LadderManager.cpp as much as possible

Reply:
Agreed.
I'll fix this up if you guys want.